packages/legend-studio/src/stores/editor-state/element-editor-state/mapping/MappingExecutionState.ts

```diff
@@ -370,10 +370,14 @@ export class MappingExecutionRelationalInputDataState extends MappingExecutionIn
 
   get runtime(): Runtime {
     const datasourceSpecification = new LocalH2DatasourceSpecification();
-    datasourceSpecification.setTestDataSetupSqls(
-      // NOTE: this is a gross simplification of handling the input for relational input data
-      [this.inputData.data],
-    );
+    if (this.inputData.inputType === RelationalInputType.CSV) {
```

Comment:
Is this if statement correct? seems like if its csv we are setting sql?

Reply:
made a mistake here, would be changing it